webpack/assets/javascripts/react_app/components/common/forms/ForemanForm/ForemanForm.js

```diff
@@ -29,6 +29,7 @@ const ForemanForm = props => (
     initialValues={props.initialValues}
     validationSchema={props.validationSchema}
     isInitialValid={isInitialValid}
+    enableReinitialize={props.enableReinitialize}
```

Comment:
Would it make sense to instead make this `true` so if the initial values change the form will always reset?

Reply:
Usually initial values are given by a constant and (I guess?) are not meant to be changed. But because `Formik` allows that, I found this pretty useful to write a form easily.

Well, if it wont break anything we have already and if no one from @theforeman/ui-ux has nothing against, I'd switch to `true` by default.

Reply:
I'm in favor of keeping this false by default, as you mentioned initialized value shouldn't be changed, so IMHO, this prop should remain controlled by the consumer.